services/rsources/handler_test.go

```diff
@@ -1585,7 +1585,7 @@ func newDBResource(pool *dockertest.Pool, networkId, hostname string, params ...
 	internalDSN := fmt.Sprintf("postgres://%[1]s:%[2]s@%[3]s:5432/%[4]s?sslmode=disable", username, password, hostname, database)
 	var (
 		db  *sql.DB
-		dsn = fmt.Sprintf("host=localhost port=%[1]s user=%[2]s password=%[3]s dbname=%[4]s sslmode=disable", port, username, password, database)
+		dsn = fmt.Sprintf("host=localhost port=%[1]s user=%[2]s password=%[3]s dbname=%[4]s sslmode=disable options='-c idle_in_transaction_session_timeout=300000' ", port, username, password, database)
```

Comment:
should we call `GetConnectionString` method here to get connection string?

Reply:
That is what I plan to do with using rudder-go-kit for the db resource. It will take some time to refactoring things here, so I would like to do them in separate PR.